packages/rpc-graphql/src/loaders/__tests__/transaction-loader-test.ts

```diff
@@ -55,7 +61,9 @@ describe('account loader', () => {
             const source = /* GraphQL */ `
                 query testQuery($signature: Signature!) {
                     transaction(signature: $signature) {
-                        slot
+                        slot {
```

Comment:
Doesn't this need to be renamed to `block` then, to reflect the new return type?